<launch>
  <arg name="JOY_CONFIG" default="$(find jsk_teleop_joy)/configs/padkontrol.yaml"/>
  <arg name="HEAD_PUB_BUTTON" default="0"/>
  <arg name="LHAND_PUB_BUTTON" default="1"/>
  <arg name="RHAND_PUB_BUTTON" default="2"/>

  <group ns="jsk_image_joy">
    <remap from="/joy" to="/interactive_control_joy"/>
    <node pkg="jsk_teleop_joy" type="midi_config_player.py"
	  name="interactive_control_joy" output="screen"
	  args="$(arg JOY_CONFIG)">
    </node>
    <node pkg="jsk_pr2_startup" type="image_snapshot_joy.py"
	  name="image_snapshot_joy" output="screen">
      <param name="head_pub_button" value="$(arg HEAD_PUB_BUTTON)"/>
      <param name="lhand_pub_button" value="$(arg LHAND_PUB_BUTTON)"/>
      <param name="rhand_pub_button" value="$(arg RHAND_PUB_BUTTON)"/>
    </node>
  </group>
</launch>
